docs/partners/integration_guide.md

```diff
@@ -2,40 +2,45 @@
 title: Integration Methods
 ---
 
-We propose two different integration methods for different use cases. You can use the one that fits you better.
+There are two methods of integration, each used for different purposes. 
```

Comment:
I'd keep: `we propose` here. Or something around: `we are just proposing this, but if you come with a new idea, we are open to integrating in a different way`

Reply:
Ahh makes sense, I was interpreting it differently. Fixed